### PR TITLE
feat(ModalFooterWizard): wizard modal footer

### PR DESCRIPTION
**Pull request by YossiSaadi**
### ModalFooterWizard

image

https://monday.monday.com/boards/3532714909/pulses/7360993682

### CODE REVIEW

File: packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.primary {
+  order: 2;
```
Comment (talkor) [inline packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss:2]: Why are you controlling the order?

File: packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+.secondary {
+  order: 1;
+}
+
+.stepDots {
+  position: absolute;
+  left: 50%;
+  translate: -50%;
```
Comment (talkor) [inline packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss:12]: transform: translate?

File: packages/core/src/components/Steps/StepsGalleryHeader.tsx [hunk 3]
```diff
@@ -9,7 +10,7 @@ export interface StepsGalleryHeaderProps extends VibeComponentProps {
   activeStepIndex: number;
   stepsCount: number;
   onChangeActiveStep: (e: React.MouseEvent, stepIndex: number) => void;
-  stepDescriptionFunc: (stepIndex: number) => string;
+  stepDescriptionFunc?: (stepIndex: number) => string;
```
Comment (talkor) [inline packages/core/src/components/Steps/StepsGalleryHeader.tsx:13]: I'm not really fond of this name

**Review by talkor — APPROVED** [review]

File: packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss (hunk 1 above)
Comment (YossiSaadi) [reply packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss:2]: > it is a bad practice, but design wants the primary to be first tabbed order
and using positive tabindex is even a worse practice
so I've manipulated the render order

File: packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss (hunk 2 above)
Comment (YossiSaadi) [reply packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss:12]: it works basically, but you are right, it is not supported enough (104 according to caniuse)

File: packages/core/src/components/Steps/StepsGalleryHeader.tsx (hunk 3 above)
Comment (YossiSaadi) [reply packages/core/src/components/Steps/StepsGalleryHeader.tsx:13]: lol
this should be re-written, I don't like the way this component works

**Pull request merged**